### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by balajigaikwad**
Assignment1

### CODE REVIEW

**Comment by roshantrivedi** [comment]
All looks good.

File: Assignment1_Notes [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Q: SPA
+A: In single page application we load only single document and there after we can update that page without refreshing the page, using JS API. 
+This gives better performance and good UI experience example Netflix. 
+However, there are few downside as well like SEO, state management and routing.
+Q: Emmet
+A: An free add on text editor tool, which has shortcut code snippets for long pieces of code.
+Q: CDN
+A: Content Delivery Network. It is a network of servers to distribute content all over the world.  A user might have a replica server of origin content stored server. This helps in quicker access to the resources.
+Q: Use of cross origin for adding external scripts
+A: Cross origin attribute allows CORS (Cross origin resource sharing).
```
Comment (Pujarini) [inline Assignment1_Notes]: Can you explain CORS in more brief please. https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const heading2 = React.createElement("h1", null, "Hello world using React");
+const root2 = ReactDOM.createRoot(document.getElementById("root2"));
```
Comment (Pujarini) [inline src/App.js:2]: naming of variables is not good. Please avoid digits in variables. Keep it root only. variables should define the action  it is used for.

**Review by Pujarini — COMMENTED** [review]
Looks good. Please check one comment.

**Review by ap221882 — APPROVED** [review]

File: Assignment1_Notes (hunk 1 above)
Comment (balajigaikwad) [reply Assignment1_Notes]: Some more info added in the file.

**Pull request merged**